src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -85,17 +84,28 @@ public HttpApiHandler(DurableTaskExtension config, ILogger logger)
         // /orchestrators/{functionName}/{instanceId?}
         private static TemplateMatcher GetStartOrchestrationRoute()
         {
-            var defaultRouteValues = RouteValueDictionary.FromArray(new KeyValuePair<string, object>[] { new KeyValuePair<string, object>(InstanceIdRouteParameter, string.Empty) });
+            var defaultRouteValues = RouteValueDictionaryFromArray(new KeyValuePair<string, object>[] { new KeyValuePair<string, object>(InstanceIdRouteParameter, string.Empty) });
             return new TemplateMatcher(TemplateParser.Parse($"{OrchestratorsControllerSegment}{{{FunctionNameRouteParameter}}}/{{{InstanceIdRouteParameter}?}}"), defaultRouteValues);
         }
 
         // /entity/{entityId}/{entityKey?}
         private static TemplateMatcher GetEntityRoute()
         {
-            var defaultRouteValues = RouteValueDictionary.FromArray(new KeyValuePair<string, object>[] { new KeyValuePair<string, object>(EntityKeyRouteParameter, string.Empty) });
+            var defaultRouteValues = RouteValueDictionaryFromArray(new KeyValuePair<string, object>[] { new KeyValuePair<string, object>(EntityKeyRouteParameter, string.Empty) });
             return new TemplateMatcher(TemplateParser.Parse($"{EntitiesControllerSegment}{{{EntityNameRouteParameter}}}/{{{EntityKeyRouteParameter}?}}"), defaultRouteValues);
         }
 
+        private static RouteValueDictionary RouteValueDictionaryFromArray(KeyValuePair<string, object>[] values)
+        {
+            var routeValueDictionary = new RouteValueDictionary();
+            foreach (var pair in values)
+            {
+                routeValueDictionary[pair.Key] = pair.Value;
+            }
+
+            return routeValueDictionary;
+        }

```

Comment:
I assume this code was copied from somewhere. If so, we should add a comment indicating this. Or perhaps just mention why we can't call `RouteValueDictionary.FromArray()`.

Reply:
`RouteValueDictionary.FromArray()` doesn't exist in the `Microsoft.AspNetCore.Routing` we use for Functions V1.